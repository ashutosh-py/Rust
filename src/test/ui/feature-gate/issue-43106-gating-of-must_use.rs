// The non-crate level cases are in issue-43106-gating-of-builtin-attrs.rs.

#![must_use = "4200"]
//~^ ERROR cannot determine resolution for the attribute macro `must_use`

fn main() {}
