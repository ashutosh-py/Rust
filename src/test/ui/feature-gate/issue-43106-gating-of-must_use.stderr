error: cannot determine resolution for the attribute macro `must_use`
  --> $DIR/issue-43106-gating-of-must_use.rs:3:4
   |
LL | #![must_use = "4200"]
   |    ^^^^^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: aborting due to previous error

